modules/peers.js

```diff
@@ -358,8 +359,8 @@ __private.insertSeeds = function(cb) {
 __private.dbLoad = function(cb) {
 	let updated = 0;
 	library.logger.trace('Importing peers from database');
-	library.db.peers
-		.list()
+	library.storage
+		.get({}, { limit: 10000 }) // @TODO: Arbitrary limit set for now. Base issue should be addressed in storage module for this cases
```

Comment:
It's not valid call. `library.storage.get`, which entity you are calling here? 

It should be `library.storage.entities.Peer`

Reply:
oh because in init I'm doing: `storage: scope.storage.entities.Peer,` do you prefer for it to be more explicit?

Reply:
renamed storage to peerStorage